cdap-docs/developer-manual/source/cloud-runtimes/example/index.rst

```diff
@@ -10,5 +10,125 @@
+All of these steps can be performed in the GCP Console.
+
+  - Enable Cloud Dataproc
+  - Create a service account key that has permission to create Cloud Dataproc clusters and view Google Compute Engine resources
+  - Ensure there is an ingress VPC firewall rule that allows tcp connections on port 443 (https)
+  - Create a Google Cloud Storage (GCS) bucket and upload a test file
+
+After you have set up your GCP account, add the service account key as a key in the CDAP secure store.
+This must be done using the :ref:`Secure Storage RESTful API <http-restful-api-secure-storage>`.
+For example, to add a secure key named `gcp-key` using curl::
+
+  $ curl localhost:11015/v3/namespaces/default/securekeys/gcp-key -X PUT -d '{
+    "properties": {},
+    "description": "gcp service account key",
+    "data": "{  \"type\": \"service_account\", \"private_key\": \"...\", ... }"
+  }'
+
+The value of the data field is the content of your GCP service account key in string form. 
+
+Create a Profile
+----------------
+
+To create a profile, navigate to the Administration page using the menu at the top right of the CDAP UI.
+From this page, you can view the list of system profiles.
+
+.. figure:: /_images/cloud-runtimes/create-system-profile.png
+  :figwidth: 100%
+  :width: 800px
+  :align: center
+  :class: bordered-image
+
+Click on the button to create a new profile. 
+
+.. figure:: /_images/cloud-runtimes/provisioners.png
+  :figwidth: 100%
+  :width: 800px
+  :align: center
+  :class: bordered-image
+
+Select the Google Cloud Dataproc provisioner.
+
+.. figure:: /_images/cloud-runtimes/provisioner-configure.png
+  :figwidth: 100%
+  :width: 800px
+  :align: center
+  :class: bordered-image
+
+Fill in all configuration fields. For the service account key, specify the name of the secure key that
+was added in the setup part of this walk-through.
+
+Create a Test Pipeline
+----------------------
+
+Click on the green plus button and create a new batch pipeline. Our test pipeline will be a very simple
+one that simply reads a file on GCS and writes to an output directory on GCS. Begin by adding a File source
+and a File sink from the left panel. Connect the source to the sink.
+
+.. figure:: /_images/cloud-runtimes/simple-pipeline-overview.png
+  :figwidth: 100%
+  :width: 800px
+  :align: center
+  :class: bordered-image
+
+Configure the source to read a file from the GCS bucket that you set up earlier.
+
+.. figure:: /_images/cloud-runtimes/simple-pipeline-input.png
+  :figwidth: 100%
+  :width: 800px
+  :align: center
+  :class: bordered-image
+
+Configure the sink to write to a directory that does not yet exist.
+
+.. figure:: /_images/cloud-runtimes/simple-pipeline-output.png
+  :figwidth: 100%
+  :width: 800px
+  :align: center
+  :class: bordered-image
+
+Deploy the pipeline by clicking on the `Deploy` button near the top right of the screen.
```

Comment:
Change to:
To deploy the pipeline, click the `Deploy` button near the top right of the screen, which opens the pipeline overview page. In this page, configure the pipeline to use the profile that you created earlier.